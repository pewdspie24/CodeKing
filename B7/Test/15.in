52
9 1317114160
60533222 939530493
796172026 599384757
168394908 340723450
198364474 932309315
890489835 235179929
405734262 211450148
263854723 309632639
600647222 343528181
210236246 150960280
6 3363500814
480392790 345374442
738957583 902981317
973124655 196348748
500023138 345236773
927751912 711003600
861388635 881108755
1 3083510070
815395500 285191834
9 2468640752
303807190 425619442
173535477 333951648
715508086 990809755
492342624 69507623
875176633 268465859
858518446 262811444
478724195 850845049
471561569 51778872
463600833 405736344
5 584422488
813314116 918950923
113103278 433847553
676372622 847301776
564806324 46216339
488609190 763175564
1 3568268698
343208318 557046232
4 1126957604
164708628 479177661
513629040 523422718
808470235 468225220
775998764 414466860
8 4119239183
918236040 893627702
373094100 91588340
691736102 649992972
808989904 698174077
563855585 281693147
802059765 373940425
953945372 526421532
253262508 795737615
4 3615189638
221614366 372073621
127622599 977981040
207770416 360194828
190796319 993757950
1 1625058151
751338151 370976538
5 1346943215
79568934 534617473
55866830 647572844
87714166 786325861
984937995 365522596
111320999 268397465
1 4001149626
272772782 138509492
9 220447536
913074128 977772618
108913713 821615667
642730170 463806626
148271342 263558231
118562292 209058987
797165165 110944292
506139861 603419466
304292227 551264644
603664297 489268715
10 3484002904
307714729 176997150
624209623 149294714
663214351 721536010
463437600 413165668
372305908 31318032
770377356 147799775
454464685 301783752
470007991 389067264
918108553 126638181
285989824 664853061
8 3882429610
107193477 613463275
485193691 142735710
277427234 979078831
611371890 200110969
927590160 11073103
999841975 552219385
319634087 441737596
114146650 84021815
10 3370818820
416005359 153693630
212096093 965001123
384738559 560422645
734466737 177125074
399752190 154029329
224007127 158868209
200604192 956341985
698544544 263340040
716661423 735552484
250982823 546555003
1 1098316815
620826109 905024196
3 243406512
990085424 218284868
34221044 395833982
398963804 721326513
10 1512188081
589420319 472069099
19601631 360888420
895204772 23722150
199696292 542473071
292934845 439210844
199254877 497973447
803231821 128188773
604175126 586537623
931272203 836004468
489082004 349140282
1 2946738770
581667103 915752381
1 2752286943
648042487 288720243
1 3211236370
250186807 803046835
5 1733160274
107746423 53478056
993067743 873966680
164559428 824524859
209572075 18574755
259396533 394233845
4 584489619
880122088 421448293
341897793 761669413
244029145 389891576
604456774 897575991
10 2664718865
106091105 883110704
79806015 534853010
715609724 265213908
387431652 801525602
944771518 411034583
105420713 328867583
883565107 195066086
176398667 986921067
9641792 26148501
659028079 164091613
4 1495609843
497835819 293925708
85085977 218181755
535459876 768647754
323736301 554280859
7 79954032
652041073 872508228
174567458 85750955
881702810 374988542
794320523 55086022
645665403 915723738
735445072 208373188
736703887 889194854
3 730997779
370756921 705871902
837468329 822783781
719715980 110123039
3 2165431413
241534671 214536394
411060524 696962952
844884268 216429699
7 3825367367
717730478 206354718
667391331 594735744
442373832 257457036
38554347 377297149
71655473 854961182
718523069 383699352
35141497 168888171
1 2592000123
708935987 154424222
10 1031416084
570846565 613547958
299541360 275347239
831003341 343558397
965885274 231006113
647079407 436126000
292918456 922450336
12795019 131869765
720329597 851239166
168494967 144532255
700819410 102540240
4 3840033121
82751411 256241251
844426576 923441085
462474138 395331207
948067615 260702649
6 420351656
148882065 976319063
975338678 46385030
234099343 146518238
814513471 323873756
44040265 745003267
622154571 401189133
5 3149865049
882042915 450600773
550616411 119165853
395704116 148719201
725394872 338296058
657602122 131404513
2 2741522144
618690878 108327305
52910440 295789042
9 3670367049
270668662 39641508
67444323 468262045
765583593 135621661
869936188 27118974
581097842 292256738
347895097 171978135
349791453 348057885
444142293 77664675
714896880 237913604
9 3876207170
730532917 331119052
956977738 550358330
951998600 378745786
947198315 377887351
145340938 813749241
964559449 53382213
621263433 916448686
864194474 307501021
322406989 488093561
4 3638076649
708616037 288091596
77302109 647702150
334841760 234372957
619074356 789725923
5 2138701658
433249591 296590795
874836946 391459672
524971619 509992440
583135310 98026644
364106466 530865480
5 2477271893
401094137 100048102
301337619 230697899
626183005 534813257
283367785 178839273
36032216 263896574
6 405771153
173330927 176210468
385627124 998702863
139707531 970840224
43816097 683215690
828352197 86309041
969384971 864388494
2 3770017184
923401168 94292199
796061284 887387197
7 1145355585
277810639 550923108
354333311 726311052
130914627 295760336
965546305 627931023
1192320 464919778
568274088 291165499
273941113 358367694
5 1603060205
801496967 862991050
947541331 585768794
417291600 153354660
808124366 11150612
592180415 362185445
9 3527677315
840234084 822314776
396852247 837792629
643706195 605022582
129717213 437002912
149070000 744926947
885714595 795357710
972994494 226039543
518304490 378425543
441470490 147035914
6 596773212
89785752 860824207
815095838 759220379
400096424 148142251
177168835 478832415
365233396 23438632
615402948 18785356
7 2893375679
704463765 484885846
352100732 669486564
234009593 58662134
728045255 463472838
843226099 570403915
43508069 634843396
579645614 80492744
3 2472328449
898147139 50665275
798858522 400508149
918276739 792076400
6 511515853
116058331 65779167
591724217 973750216
562402639 412171198
660903752 416449727
773277534 704512844
738669123 108151191
4 1655792214
750223609 991834904
100138603 273961117
875519683 925956047
653284121 128359956
3 1941483297
303186039 189294683
416541017 804341172
566801645 616963121
