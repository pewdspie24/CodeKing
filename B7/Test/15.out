2
5
6
4
1
MISSION FAILED!
2
9
MISSION FAILED!
4
2
29
1
5
5
6
MISSION FAILED!
1
2
MISSION FAILED!
7
MISSION FAILED!
5
1
4
6
1
1
4
8
5
2
6
1
6
6
5
5
8
4
8
1
5
2
2
5
1
6
3
1
3
14
